dashboard/app/models/user.rb

```diff
@@ -114,6 +114,9 @@ class User < ActiveRecord::Base
     last_seen_school_info_interstitial
     ui_tip_dismissed_homepage_header
     ui_tip_dismissed_teacher_courses
+    oauth_refresh_token
+    oauth_token
+    oauth_token_expiration
```

Comment:
OAuth token expiration is the timestamp for when the `ouath_token` will no longer work and should be refreshed via `oauth_refresh_token`.  This is so that we only refresh the access token when necessary.

It's not clear to me what bad things happen if we abuse the refresh token too much, but all the docs say to only refresh the token when necessary.